src/pages/Login.js

```diff
@@ -1,78 +1,78 @@
+// imports
 import React, { useState } from 'react';
 import axios from 'axios';
 import jwt_decode from 'jwt-decode';
 import { Redirect } from 'react-router-dom';
-import setAuthToken from '../utilities/setAuthToken'
+import setAuthToken from '../utilities/setAuthToken';
+import logo from '../assets/images/hscLogo21.png';
+//import Footer from '../components/Footers/Footer.js';
 
 import '../styles/login.styles.css'
 
-
+// storing database url in environment
 const { REACT_APP_DATABASE_URL } = process.env
 
 const Login = (props) => {
+
+    // state variables
     const [email, setEmail] = useState('');
     const [password, setPassword] = useState('');
     const [redirect, setRedirect] = useState(false)
 
-
+    // set email based on event
     const handleEmail = (e) => {
         setEmail(e.target.value);
     }
 
+    // set password based on event
     const handlePassword = (e) => {
         setPassword(e.target.value);
     }
 
-    const handleSubmit = (e) => {
+    // handle submit function
+    const handleSubmit = async (e) => {
         console.log(`>>>> inside handleSubmit func`)
         e.preventDefault()
         const userData = { email, password }
 
-        axios.post(`${REACT_APP_DATABASE_URL}/users/login`, userData)
-            .then(response => {
-                console.log(`>>>>inside handleSubmit reponse block`)
-
-                console.log(response)
-                const { token } = response.data
-                //save token to localStorage
-                localStorage.setItem('jwtToken', token)
-                //set token to header
-                setAuthToken(token)
-                //decode token to get user data
-                const decoded = jwt_decode(token)
-                props.nowCurrentUser(decoded)
-                setRedirect(true)
-            })
-            .catch(error => {
-                console.log(`>>>> error logging in ${error}`)
-                alert('incorrect email or password! please try again')
-            })
+        try 
+        {
+            const response = await axios.post(`${REACT_APP_DATABASE_URL}/users/login`, userData)
+            console.log(`>>>>inside handleSubmit reponse block`)
+            const token = await response.data
+            await localStorage.setItem('jwtToken', token)
+            setAuthToken(token)
+            const decoded = jwt_decode(token)
+            props.nowCurrentUser(decoded)
+            setRedirect(true) 
+        }
+        catch(error)
+        {
+            console.log(`>>>> error in handleSubmit func: ${error}`)
```

Comment:
We may want to revisit how we're handling this error, if an error (lets say a 422 or 500) error occurs the user will be told its because of incorrect credentials